.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
Hm... should this file be in the .gitignore? Not sure if it matters since I don't use VS/VS Code...